airbyte-workers/src/main/java/io/airbyte/workers/temporal/sync/RouteToTaskQueueActivityImpl.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.workers.temporal.sync;
+
+import io.airbyte.config.Configs;
+import io.airbyte.workers.temporal.TemporalJobType;
+import java.util.UUID;
+
+public class RouteToTaskQueueActivityImpl implements RouteToTaskQueueActivity {
+
+  // To be replaced by proper Routing Service. For now, our MVP supports a single external Data Plane
+  private static final String MVP_DATA_PLANE_TASK_QUEUE = "MVP_DATA_PLANE";
+
+  private final Configs configs;
+
+  public RouteToTaskQueueActivityImpl(final Configs configs) {
+    this.configs = configs;
+  }
+
+  @Override
+  public String routeToTaskQueue(final UUID connectionId) {
+    if (configs.connectionIdsForDataPlane().contains(connectionId.toString())) {
```

Comment:
@cgardens previous discussion from parent PR:
![image](https://user-images.githubusercontent.com/7798112/186286305-83b90e02-c11e-486c-88f2-237ce72d0f51.png)

Reply:
Parker, were you going to add a comment here?

Reply:
Yes, I added the `RouterService` class that Jimmy recommended, with a comment explaining how this part works.